### ISSUE TITLE
getConceptsInCohort function's sql appears to have a BUG

### ISSUE THREAD

**Issue by gowthamrao** [OHDSI/CohortDiagnostics#877]
Missing group by here

https://github.com/OHDSI/CohortDiagnostics/blob/5567b89bd29ae84d9719e0805b1f3ee8b516dd9c/inst/shiny/DiagnosticsExplorer/R/Results.R#L293-L306

**Comment by gowthamrao** [comment]
Should be
```

            	UNION

            	SELECT c.database_id,
            		c.cohort_id,
            		c.source_concept_id concept_id,
            		1 source_concept_id,
            		max(c.concept_subjects) concept_subjects,
            		sum(c.concept_count) concept_count
            	FROM @results_database_schema.included_source_concept c
            	WHERE c.cohort_id = @cohort_id
            		AND c.database_id IN (@database_ids)
            	GROUP BY c.database_id,
            		c.cohort_id,
            		c.source_concept_id
            	) concepts
```

**Comment by azimov** [comment]
Already fixed in the develop branch, no fixes will be made to 3.0.x shiny code because there are no tests and data will be compatible with the 3.1.0 release
